tests/performance_tests/performance_test_basic.py

```diff
@@ -204,22 +206,34 @@ def fileOpenMode(self, filePath) -> str:
 
     def queryBlockTrxData(self, node, blockDataPath, blockTrxDataPath, startBlockNum, endBlockNum):
         for blockNum in range(startBlockNum, endBlockNum):
-            block = node.processUrllibRequest("trace_api", "get_block", {"block_num":blockNum}, silentErrors=False, exitOnError=True)
+            if self.clusterConfig.isNodeosv2:
+                block = node.processUrllibRequest("chain", "get_block", {"block_num_or_id":blockNum}, silentErrors=False, exitOnError=True)
+            else:
+                block = node.processUrllibRequest("trace_api", "get_block", {"block_num":blockNum}, silentErrors=False, exitOnError=True)
```

Comment:
Should use similar approach as you did in `Node.py` where these could be lambda set in the `__init__` and used here.